README.md

```diff
@@ -1,5 +1,12 @@
 # clang-tidy pull request comments
-![clang-tidy-8 support] ![clang-tidy-9 support] ![clang-tidy-10 support] ![clang-tidy-11 support] ![clang-tidy-12 support]
+
+[![Clang-Tidy 8 support](https://img.shields.io/badge/clang--tidy-8-brightgreen)](https://releases.llvm.org/8.0.0/tools/clang/tools/extra/docs/clang-tidy/index.html)
+[![Clang-Tidy 9 support](https://img.shields.io/badge/clang--tidy-9-brightgreen)](https://releases.llvm.org/9.0.0/tools/clang/tools/extra/docs/clang-tidy/index.html)
+[![Clang-Tidy 10 support](https://img.shields.io/badge/clang--tidy-10-brightgreen)](https://releases.llvm.org/10.0.0/tools/clang/tools/extra/docs/clang-tidy/index.html)
+[![Clang-Tidy 11 support](https://img.shields.io/badge/clang--tidy-11-brightgreen)](https://releases.llvm.org/11.0.0/tools/clang/tools/extra/docs/clang-tidy/index.html)
+[![Clang-Tidy 12 support](https://img.shields.io/badge/clang--tidy-12-brightgreen)](https://releases.llvm.org/12.0.0/tools/clang/tools/extra/docs/clang-tidy/index.html)
+[![Clang-Tidy 13 support](https://img.shields.io/badge/clang--tidy-13-brightgreen)](https://releases.llvm.org/13.0.0/tools/clang/tools/extra/docs/clang-tidy/index.html)
+[![Clang-Tidy 14 support](https://img.shields.io/badge/clang--tidy-14-brightgreen)](https://releases.llvm.org/14.0.0/tools/clang/tools/extra/docs/clang-tidy/index.html)
```

Comment:
Could you please add the links for the "shields" in the bottom of the document instead where the existing links can be found?

Reply:
So you want to remove these badges from here and move them to the "Supported clang-tidy versions" section below instead?

Reply:
In the bottom of `README.md` there's the following snippet:

```
[clang-tidy-8 support]: https://img.shields.io/badge/clang--tidy-8-green
[clang-tidy-9 support]: https://img.shields.io/badge/clang--tidy-9-green
[clang-tidy-10 support]: https://img.shields.io/badge/clang--tidy-10-green
[clang-tidy-11 support]: https://img.shields.io/badge/clang--tidy-11-green
[clang-tidy-12 support]: https://img.shields.io/badge/clang--tidy-12-green
```

What I am suggesting is to add `clang-tidy-13 support` and `clang-tidy-14 support` down there instead. The links to the respective llvm release could remain here I guess and are a nice addition. :100: